_config.yml

```diff
@@ -16,3 +16,7 @@ defaults:
       weight: 100   # Used to sort navbar items. Lower weight goes higher on the list.
 
 highlighter: rouge
+
+# This is the best we can get to avoid the 
```

Comment:
This is the best I could get to not to load the build with useless files from the submodule. I tried every glob expression to only include the `yaml` and `proto` but I failed to get a decent result.